ledger/ledger-api-bench-tool/src/main/scala/com/daml/ledger/api/benchtool/metrics/MetricsSet.scala

```diff
@@ -88,6 +37,19 @@ object TransactionMetrics {
       objectives = objectives,
     )
 
+  def activeContractsMetrics: List[Metric[GetActiveContractsResponse]] =
+    List[Metric[GetActiveContractsResponse]](
+      CountRateMetric.empty[GetActiveContractsResponse](
+        countingFunction = _.activeContracts.length
+      ),
+      TotalCountMetric.empty[GetActiveContractsResponse](
+        countingFunction = _.activeContracts.length
+      ),
+      SizeMetric.empty[GetActiveContractsResponse](
+        sizingFunction = _.serializedSize.toLong
+      ),
+    )
```

Comment:
I haven't found any record time field in the `GetActiveContractsResponse` which makes it impossible to use the `ConsumptionSpeedMetric` and `DelayMetric` for the ACS stream.

Reply:
ACS is not necessarily received in a chronological order, so it would not make much sense.